### ISSUE TITLE
Add the missing skip to main content link 

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#206]
@krozett There's no skip to main content link for keyboard only users.

Accessibility success criterion: https://www.w3.org/TR/WCAG21/#bypass-blocks

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
